functions-methods.js

```diff
@@ -33,4 +79,27 @@
 // checkEmailValidity("tessmellink@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
-// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
\ No newline at end of file
+// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+function checkEmailValidity(email){
+    if (email.includes('@') && !email.includes(',') && email.slice(-1) !== '.'){
+        console.log('true - want @ en punt op de juiste plek');
+    }
+    if(!email.includes('@')) {
+        console.log('false - want geen @');
+    }
+    if (email.includes(',')){
+        console.log('false - want er staat een komma in');
+    }
+    if (email.slice(-1) === '.'){
+        console.log('false - want de punt mag niet als laatst');
+    }
```

Comment:
Het was niet perse de bedoeling dat je de reden mee zou loggen. Als je dat niet doet dan zou je de functie drastisch kunnen versimpelen:
```suggestion
    return email.includes('@') && !email.includes(',') && email.slice(-1) !== '.';
```